lib/widgets/buttons.dart

```diff
@@ -0,0 +1,453 @@
+  final Color buttonColor;
+  final Color textColor;
+  final VoidCallback onPressed;
+
+  LongButton(this.text, this.buttonColor, this.textColor, this.onPressed);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 224,
+      height: 64,
+      decoration: BoxDecoration(
+          color: buttonColor,
+          borderRadius: BorderRadius.all(Radius.circular(5))),
+      child: TextButton(
+        onPressed: () => onPressed(),
+        child: Text(
+          text,
+          style: TextStyle(color: textColor),
+        ),
+      ),
+    );
+  }
+}
+
+class ShortButton extends StatelessWidget {
+  final String text;
+  final Color buttonColor;
+  final Color textColor;
+  final VoidCallback onPressed;
+
+  ShortButton(this.text, this.buttonColor, this.textColor, this.onPressed);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 128,
+      height: 64,
+      decoration: BoxDecoration(
+          color: buttonColor,
+          borderRadius: BorderRadius.all(Radius.circular(5))),
+      child: TextButton(
+        onPressed: () {
+          onPressed();
+        },
+        child: Text(
+          text,
+          style: TextStyle(color: textColor),
+        ),
+      ),
+    );
+  }
+}
+
+class SquareButton extends StatelessWidget {
+  final String text;
+  final Color buttonColor;
+  final Color textColor;
+  final VoidCallback onPressed;
+
+  SquareButton(this.text, this.buttonColor, this.textColor, this.onPressed);
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 54,
+      height: 54,
+      decoration: BoxDecoration(
+          color: buttonColor,
+          borderRadius: BorderRadius.all(Radius.circular(5))),
+      child: TextButton(
+          onPressed: () {
+            onPressed();
+          },
+          child: Text(
+            '$text',
+            style: TextStyle(color: textColor),
+          )),
+    );
+  }
+}
+
+class ShrinkingButton extends StatelessWidget {
+  final String text;
+  final Color buttonColor;
+  final Color textColor;
+  final VoidCallback onPressed;
+
+  ShrinkingButton(this.text, this.buttonColor, this.textColor, this.onPressed);
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 41,
+      height: 54,
+      decoration: BoxDecoration(
+          color: buttonColor,
+          borderRadius: BorderRadius.all(Radius.circular(5))),
+      child: TextButton(
+          onPressed: () {
+            onPressed();
+          },
+          child: Text(
+            '$text',
+            style: TextStyle(color: textColor),
+          )),
+    );
+  }
+}
+
+class DropDown extends StatefulWidget {
+  @override
+  _DropDownState createState() => _DropDownState();
+}
+
+class _DropDownState extends State<DropDown> {
+  int _value = 1;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 185,
+      height: 54,
```

Comment:
height 56

for the width, im not sure if it should really be 185 (or it expands) but that can still work